aviary/variable_info/variables.py

```diff
@@ -155,7 +155,7 @@ class Design:
         FINENESS = 'aircraft:design:fineness'
         FIXED_EQUIPMENT_MASS = 'aircraft:design:fixed_equipment_mass'
         FIXED_USEFUL_LOAD = 'aircraft:design:fixed_useful_load'
-        IJEFF = 'ijeff'
+        IJEFF = 'aircraft:design:ijeff'
```

Comment:
This one is intentional
```suggestion
        IJEFF = 'ijeff'
```

Reply:
@crecine 
Without this modification "ieff" becomes a subheading and since this variable is in the middle of the "design" section it takes the rest of the design variables into its container. Would it be better to move this variable to later in the variable list or is there a better option?

Reply:
if we need this change to prevent that issue, that's fine. But since we don't need to set this value, excluding it from the GUI would also work